products/ceqr_survey/models/intermediate/int__all_buffers.sql

```diff
@@ -0,0 +1,18 @@
+WITH all_buffers AS (  -- noqa: disable=all
+{{ dbt_utils.union_relations(
+    relations=[
```

Comment:
This list of refs matches what we'd want to export to the fgdb, which makes me wonder whether we could make it a seed, cc @damonmcc

Reply:
Looks clean! Nice

Reply:
true. or maybe since these are table names that will definitely be in the a yaml file, we could use a [tag](https://docs.getdbt.com/reference/resource-configs/tags)? something like:
```
models:
  - name: int__elevated_railways
    config:
      tags: ["buffers"]
```

I'm not sure if `dbt_utils.union_relations` can select relations based on tags though

Reply:
oh and can't forget about the `metadata.yml` ideas where we imagined declaring packaging details like "which tables should be in the FGDB"